LabExT/Movement/PathPlanning.py

```diff
@@ -247,11 +247,17 @@ def __init__(
             y=self.y_coords[self.current_idx[1]],
             z=self.start_coordinate.z)
 
-        # field target, note: z-coordinate remains unchanged
-        self.target_coordinate = ChipCoordinate(
-            x=target_coordinate.x,
-            y=target_coordinate.y,
-            z=self.start_coordinate.z)
+        # field target
+        self.target_coordinate = target_coordinate
+
+        if not np.isclose(
+                self.start_coordinate.z,
+                self.target_coordinate.z,
+                rtol=1.e-5,
+                atol=1.e-8,):
```

Comment:
I took the default values here. Are they ok, @marcoep ?

Reply:
Don't wonder, I just removed the comma after `atol=1.e-8,`

Reply:
I would set it such that it is okay within a +-10nm range. So keep `rtol` the same, but change `atol` to something like 10e-2 (assuming we are in units of microns).

Reply:
Changed it to 10e-2. Yes all units are in um.

Reply:
ahh, sorry, meant 10e-3 (i.e. 10nm)

Reply:
okay its 10e-3 now 😄